src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustCorporateValidatorTest.java

```diff
@@ -182,12 +184,45 @@ void testErrorReportedWhenDateBecameInterestedPersonIsInTheFuture() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    @Test
+    void testNoErrorReportedWhenDateBecameInterestedPersonIsAfterTrustCreationDate() {
+        trustDataDtoList.get(0).getCorporates().get(0).setType(BeneficialOwnerType.INTERESTED_PERSON.getValue());
+        trustDataDto.setCreationDate(LocalDate.of(2000, 1,1));
```

Comment:
Shouldn't this test and the two following be checking the 'ceased date' in relation to the 'date became interested'?

Reply:
Further down the interested date is being tested against the ceased date but is only testing one scenario; I thought I'd include the creation date as well for coverage. As this is not relevant to the work for this ticket I have removed them and added a couple more tests for the ceased date instead to cover the other scenarios.

Reply:
Ah yes. That's why I was confused and because those scenarios weren't covered I thought this method was trying to do that. Latest changes look good to me.